src/transformers/audio_utils.py

```diff
@@ -460,6 +461,10 @@ def spectrogram(
         onesided (`bool`, *optional*, defaults to `True`):
             If True, only computes the positive frequencies and returns a spectrogram containing `fft_length // 2 + 1`
             frequency bins. If False, also computes the negative frequencies and returns `fft_length` frequency bins.
+        dither (`float`):
```

Comment:
You should also make sure that the docstrings is right here and in the rest of the docstrings
```suggestion
        dither (`float`, *optional*, defaults to 0.0):
```